readme.md

```diff
@@ -92,6 +92,33 @@ const fileType = require('file-type');
 })();
 ```
 
+Or from a _tokenizer_:
+
+A [_tokenizer_](https://github.com/Borewit/strtok3#tokenizer) propagates the internal read functions, 
+allowing alternative transport mechanisms, to access files, to be implemented and used.
+
+An example is [@tokenizer/http](https://github.com/Borewit/tokenizer-http) which requests data using range-requests.
+A difference with a conventional stream, is that it for example allows to only read the first 6, and the last 128 bytes.
```

Comment:
The docs in the TS file and readme should be in sync.

Reply:
I missed the API documentation. I updated that path with the Typescript definitions.